models/mealfood.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const MealFood = sequelize.define('MealFood', {
+    foodId: {type: DataTypes.INTEGER,
+             allowNull: false,
+             references: {
+               model: 'Food',
+               key: 'id'
+             }
+           },
+           
+    mealId: {type: DataTypes.INTEGER,
+             allowNull: false,
+             references: {
+               model: 'Meal',
+               key: 'id'
+             }
+           }
+
```

Comment:
It's strange that this works, but, Sequelize is a rabbit hole we don't need to go any deeper down for now 🐇

Reply:
The guide I followed for this was found at https://medium.com/@the_ozmic/how-to-create-many-to-many-relationship-using-sequelize-orm-postgres-on-express-677753a3edb5